packages/modules/chromadb/src/chromadb-container.ts

```diff
@@ -3,7 +3,7 @@ import { AbstractStartedContainer, GenericContainer, StartedTestContainer, Wait
 const CHROMADB_PORT = 8000;
 
 export class ChromaDBContainer extends GenericContainer {
-  constructor(image = "chromadb/chroma:0.4.22") {
+  constructor(image = "chromadb/chroma:0.5.0") {
```

Comment:
IIRC, this could bring a breaking change with the env var for auth and also for those relying on version 0.4.22.

Reply:
Thanks @eddumelendez, indeed Auth is a breaking change in 0.5.0

Reply:
what I meant is that existing users using default image will break test just by upgrading testcontainers, this should not happen in a patch release. So, my suggestion is to keep 0.4.22, users who want to use a different version can do it by overriding the default image.